### PR TITLE
Much improved clocks and port

### PR DESCRIPTION
**Pull request by tortellinitony**
Adds clocks, drastically improved from those on Hippie before. New *animated* sci-fi icon.
http://i.imgur.com/094v804.png

Don't like it? You can break them now.
http://i.imgur.com/zYqqNXa.png

The bar and certain areas have a special analog clock.
http://i.imgur.com/JPcsaGm.png

The two previous bugs have been squashed.

### CODE REVIEW

**Comment by tortellinitony** [comment]
I have to do something about the map

**Comment by tortellinitony** [comment]
map merger doesn't FUCKING WORK

**Comment by Pyko1** [comment]
thats because my map pr makes changes all around the map.

**Comment by tortellinitony** [comment]
Map works.

File: _maps/map_files/TgStation2/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -12441,6 +12444,13 @@
 /area/maintenance/fpmaint2)
 "axs" = (
 /obj/structure/closet/wardrobe/black,
+/obj/machinery/clock{
+	desc = "An old fashioned device used to measure time.";
+	health = 9999;
```
Comment (j-mie) [inline _maps/map_files/TgStation2/tgstation.2.1.3.dmm]: Why does the clock have 9999 health?

File: _maps/map_files/TgStation2/tgstation.2.1.3.dmm [hunk 2]
```diff
@@ -15529,6 +15560,39 @@
...
+/turf/open/floor/plasteel,
+/area/quartermaster/miningdock)
+"aGE" = (
+/obj/machinery/clock,
+/turf/open/floor/plasteel,
+/area/hallway/primary/central)
+"aGF" = (
+/obj/machinery/clock{
+	desc = "An old fashioned device used to measure time.";
+	health = 9999;
```
Comment (j-mie) [inline _maps/map_files/TgStation2/tgstation.2.1.3.dmm]: 9999 health?

File: _maps/map_files/TgStation2/tgstation.2.1.3.dmm [hunk 3]
```diff
@@ -25220,6 +25287,13 @@
 /obj/structure/window/reinforced{
 	dir = 4
 	},
+/obj/machinery/clock{
+	desc = "An old fashioned device used to measure time.";
+	health = 9999;
```
Comment (j-mie) [inline _maps/map_files/TgStation2/tgstation.2.1.3.dmm:25292]: 9999 health?

**Review by j-mie — CHANGES_REQUESTED** [review]

**Comment by tortellinitony** [comment]
no broken icon for the bar clock

**Review by j-mie — APPROVED** [review]

**Comment by j-mie** [comment]
Fix the merge conflict

**Comment by tortellinitony** [comment]
Conflicts resolved

**Pull request merged**